packages/edit-site/src/components/sidebar-edit-mode/page-panels/style.scss

```diff
@@ -36,15 +61,21 @@
 
 .edit-site-summary-field {
 	.components-dropdown {
-		flex-grow: 1;
+		width: 70%;
```

Comment:
what is "70" :)

Reply:
It's to stop this being full width:

<img width="315" alt="Screenshot 2023-09-12 at 11 06 52" src="https://github.com/WordPress/gutenberg/assets/275961/fcaa4078-f908-48b3-82c5-91d46f7468e7">